IOHexperimenter/Cpp.md

```diff
@@ -29,35 +29,39 @@ If you want to change build options, check the output of `cmake -L` or use `cmak
 
-An example of testing the random search on __OneMax__ is provided in [problem_example.h](problem_example.h). A problem class is initiated with the given instance and dimension. The following statement declares a pointer of the instance 1 of `OneMax` in dimension 10, which belongs to the `pbo` space.
+An example of testing a random search on OneMax problem is provided in [problem_example.h](https://github.com/IOHprofiler/IOHexperimenter/blob/master/example/problem_example.h).
+
+We use the following statement to declare a `problem` class of the instance 1 of onemax in dimension 10. Instances of a problem are initiated with different transformations. Details of transformations can be found in [Section 3.2](https://www.sciencedirect.com/science/article/pii/S1568494619308099) of our publication.
 ```cpp
 const auto om = std::make_shared<ioh::problem::pbo::OneMax>(1, 10);
 ```
-Similarly, the following statement declares a pointer of the instance 1 of `Sphere` in dimension 5, which belongs to the `bbob` space. 
+Similarly, the following statement declares a class of `Sphere`, which belongs to the `bbob` space. 
 ```cpp
 const auto sp = std::make_shared<ioh::problem::bbob::Sphere>(1, 5);
 ```
 
-The `meta_data()` class provides access to useful information of the problem. The following statement can output the information of the `OneMax` class.
+To evaluate solutions found by the algorithms, we can obtain the fitness values of `x` as below.
 ```cpp
-std::cout << om->meta_data() << std::endl;
+auto y = (*om)(x).at(0)
 ```
 
-To obtain the fitness of `x` on the `OneMax`, we can easily use the statement.
+In addition, the `meta_data()` class provides access to useful information about the problem. For example, we can use the following statement to output information.
 ```cpp
-auto y = (*om)(x).at(0)
+std::cout << om->meta_data() << std::endl;
 ```
 
+
+
 <a name = "using-suites"></a>
 ## Testing Suites
 
-Suites are collections of problems. Two suites, PBO for the pseudo-Boolean problems and BBOB for the continuous problems, are available in __IOHexperimenter__. An example of using suites is provided in [suite_example.h](suite_example.h).
+Suites are collections of problems. Two suites, [PBO](https://www.sciencedirect.com/science/article/pii/S1568494619308099)) for the pseudo-Boolean optimization problems and [BBOB](https://coco.gforge.inria.fr/downloads/download16.00/bbobdocfunctions.pdf) for continuous optimization problems are available in __IOHexperimenter__. An example of using suites is provided in [suite_example.h](https://github.com/IOHprofiler/IOHexperimenter/blob/master/example/suite_example.h).
```

Comment:
Maybe mention that these are built-in, but that new suites can also be created (and link to the page which has the instructions for that)